### PR TITLE
panel_11

### PR DESCRIPTION
**Pull request by erinlincoln**
- Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/panel_11.ncl
- Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/panel_11_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/panel_11_2_lg.png
- Issue #333 
image
image

### CODE REVIEW

File: Plots/Panels/NCL_panel_11.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+times = ds.time
+scale = 1000000
+chi = ds.CHI
+chi = chi / scale
+
+###############################################################################
+# Plot:
+
+
+def Plot(ax_list):
```
Comment (anissa111) [inline Plots/Panels/NCL_panel_11.py:39]: Generally, functions should be lowercase by convention. Also, the name "Plot" kind of vague, especially with the default matplotlib call also being called "plot".

File: Plots/Panels/NCL_panel_11.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    """
+    Args:
+        ax_list : list
+            list of axes to be plotted
+
+    Plots a series of contour subplots with position depending on ax_list.
```
Comment (anissa111) [inline Plots/Panels/NCL_panel_11.py:45]: Same comment as before wrt helper function docstrings

**Review by anissa111 — COMMENTED** [review]
This looks great! My only comment would be to look at matplotlib's mosaic function that's in one of its newest releases: https://matplotlib.org/stable/tutorials/provisional/mosaic.html

You can decide if you want to use it here, but it would be cool to use it on an example like this (and add it to concepts illustrated!)

You may also want to consider bumping down your contour linewidths just a hair.

**Comment by michaelavs** [comment]
Overall, this looks good! I do have one question though for both @anissa111 and @erinlincoln: When you run the code locally, what does the x-axis look like for the first plot output in the second row of projections (the row with 3 plots)? My x-axis for the far right plot in that row only has 90E on it, but it looks like from the image provided above that Erin's version has 45E and 90E on it (as it is also seen in the NCL version). If both of you have the correct x-axis labels, then I'm fine with approving this PR right now, but if not, I want to make sure there isn't something else going on to cause the differences.

My version:
image

Above version:
image

It looks like this is also rolling over into the second plot (same plot location) so I'm not sure if the x_tick elif statements are maybe catching the data and placing it like this or again if it's just something on my side for some reason.

**Comment by erinlincoln** [comment]
When I ran it locally it also looked the same as yours - it seems like the order of the plots is wrong? I can play around and see what is happening.

**Comment by erinlincoln** [comment]
They now show in the correct order on my end locally - @anissa111 @michaelavs could you confirm it is the same on your end?

**Comment by michaelavs** [comment]
> They now show in the correct order on my end locally - @anissa111 @michaelavs could you confirm it is the same on your end?

I just re-ran it locally and I see the update! I'll go ahead and look it over again and review it

File: Plots/Panels/NCL_panel_11.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        cs = axis.contour(lon,
+                          times,
+                          chi,
+                          levels=np.arange(-10, 12, 2),
+                          colors='black',
+                          linestyles="-",
+                          linewidths=.8)
+
+        # # Label the contour levels -4, 0, and 4
+        cl = axis.clabel(cs, fmt='%d', levels=[-4, 0, 4], fontsize=7)
```
Comment (michaelavs) [inline Plots/Panels/NCL_panel_11.py:80]:
```suggestion
        axis.clabel(cs, fmt='%d', levels=[-4, 0, 4], fontsize=7)
```
This doesn't need to be assigned to a variable

File: Plots/Panels/NCL_panel_11.py [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+        # Draw contour lines at levels [-10, -8, -6, -4, -2, 0, 2, 4, 6, 8, 10]
+        cs = axis.contour(lon,
+                          times,
+                          chi,
+                          levels=np.arange(-10, 12, 2),
+                          colors='black',
+                          linestyles="-",
+                          linewidths=.8)
+
+        # # Label the contour levels -4, 0, and 4
```
Comment (michaelavs) [inline Plots/Panels/NCL_panel_11.py:79]:
```suggestion
        # Label the contour levels -4, 0, and 4
```
Extra #

File: Plots/Panels/NCL_panel_11.py [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+        # Increase the lower longitude boundary
+        l_boundary += 10
+
+    # Set plot size and show figure
+    plt.gcf().set_size_inches(12, 16)
+    plt.show()
+
+
+# Define first figure using subplot_mosaic, which allows for subplots to be
+# created with custom spacing and empty plots, which are represented by .
```
Comment (michaelavs) [inline Plots/Panels/NCL_panel_11.py:152]:
```suggestion
# created with custom spacing and empty plots, represented by '.' in function
```
just an edit for clarity

**Review by michaelavs — COMMENTED** [review]
Great start on this! Just a couple of small edits, but everything looks good overall

File: Plots/Panels/NCL_panel_11.py [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+    fig: :'figure':
+        A figure defined with mosaic_subplot.
+
+    Description
+    -----------
+
+        Takes in a defined figure and plots a contour plot, where each
+        subplot progressively shifts its longitude range east. The function
+        also sets a label beneath the plot, titles and axes labels, and contour
+        labels.
```
Comment (anissa111) [inline Plots/Panels/NCL_panel_11.py:54]:
```suggestion
    """Plots a series of contour subplots with position depending on axes in
    mosaic subplot. Takes in a defined figure and plots a contour plot, where each
    subplot progressively shifts its longitude range east. The function
    also sets a label beneath the plot, titles and axes labels, and contour
    labels.

    Parameters
    ----------
    fig: :'figure':
        A figure defined with mosaic_subplot.
```

Generally, the description should go at the top and does not need its own header

**Review by anissa111 — COMMENTED** [review]
This looks almost ready! I've just left a few comments below.

**Review by anissa111 — APPROVED** [review]
I pushed a small pre-commit fix, but when all the checks pass, this is good to merge!

**Pull request merged**